app/models/miq_server/worker_management/base/monitor.rb

```diff
@@ -58,64 +47,14 @@ def sync_workers
   end
 
   def sync_from_system
-    if podified?
```

Comment:
Will these eventually have `raise NotImplementedError, "must be implemented in a subclass"` ?

Reply:
I need to do a bit of refactoring then these shouldn't need to be here at all

Reply:
Done